src/lib/metricsMgr/MetricsManager.cpp

```diff
@@ -216,6 +227,57 @@ void MetricsManager::reset(void)
 
 /* ****************************************************************************
 *
+* metricsRender - 
+*/
+static std::string metricsRender(std::map<std::string, uint64_t>& metricsMap)
+{
+  std::map<std::string, uint64_t>::iterator  it;
+  uint64_t                                   incomingTransactions = 0;
+  uint64_t                                   totalServiceTime     = 0;
+  int                                        metricsAdded         = 0;
```

Comment:
It seems this counter is not used at the end (I mean, I see the increments, but now who is using the final value).